doc/source/dev/roadmap-detailed.rst

```diff
@@ -78,8 +78,8 @@ Pythran is still an optional build dependency, and can be disabled with
 happen it must be clear that the maintenance burden is low enough.
 
 
-Use of venerable Fortran libraries
-``````````````````````````````````
+Use of Fortran libraries
+````````````````````````
```

Comment:
If we're touching this anyway, let's update the text to say that the goal is to remove all Fortran code, and reference https://github.com/scipy/scipy/issues/18566. We're now far enough along and the end goal seems within reach, so time to make it official.

Reply:
I can do that.

Reply:
It was:

>  SciPy owes a lot of its success to relying on wrapping well established
Fortran libraries (QUADPACK, FITPACK, ODRPACK, ODEPACK etc). Some of these
libraries are aging well, others less so. We should audit our use of these
libraries with respect to the maintenance effort, the functionality, and the
existence of (possibly partial) alternatives, *including those inside SciPy*.

So replace all but the first sentence with something like

> For `<reasons>`, however, we are transitioning all Fortran libraries to more maintainable
implementations; see `gh-18566 <https://github.com/scipy/scipy/issues/18566>`__.

? 

Could you fill in `<reasons>`?

Reply:
Here is how I'd rewrite this section:
```
SciPy owes a lot of its success to relying on wrapping well established
Fortran libraries (QUADPACK, FITPACK, ODRPACK, ODEPACK etc). The Fortran 77
that these libraries are written in is quite hard to maintain, and the use
of Fortran is problematic for many reasons, e.g., it makes our wheel builds
much harder to maintain, it has repeatedly been problematic for supporting
new platforms like macOS arm64 and Windows on Arm, and it is highly problematic
for Pyodide's SciPy support. Our goal is to remove all Fortran code from SciPy,
by replacing the functionality with code written in other languages. Progress
towards this goal is tracked in `gh-18566 <https://github.com/scipy/scipy/issues/18566>`__.
```

Reply:
LGTM!

Reply:
Besides `interpolate`, which is being handled in gh-22116, there is just one other mention of Fortran here, and that's in `sparse.linalg`. 

> - Fortran code not re-entrant (but we don't solve, maybe reuse from PyKrilov)

@ilayn, it looks like that conversion was already done? Should I just remove that line?

Reply:
I think the needed parts are now quite outdated. LU decomposition situation has settled as lu_solve, lu_factor is for low-level users. We wrapped quite a number of LAPACK funcs since this was written and we don't get any requests anymore as we used to. Flapack part is also done/dormant. So I think these are all done except the batch issue. 

One needed item that comes to mind for linalg is, making SciPy, ILP64 compatible (LAPACK/BLAS using 64bit integers). It actually applies to all modules but predominantly hits linalg.

 > it looks like that conversion was already done? Should I just remove that line?

Yes, all Fortran rewrites are re-entrant and thread-safe.